apps/studio/src/server/modules/user/__tests__/user.service.test.ts

```diff
@@ -0,0 +1,47 @@
+import { resetTables } from "tests/integration/helpers/db"
+import { setupUser, setUpWhitelist } from "tests/integration/helpers/seed"
+
+import { isUserDeleted } from "../user.service"
+
+describe("user.service", () => {
+  beforeAll(async () => {
+    await resetTables("User")
+    await setUpWhitelist({ email: "@example.com" })
+
+    // Setup active user
+    await setupUser({
+      name: "Active User",
+      userId: "active123",
+      email: "active@example.com",
+      phone: "12345678",
+      isDeleted: false,
+    })
+
+    // Setup deleted user
+    await setupUser({
+      name: "Deleted User",
+      userId: "deleted123",
+      email: "deleted@example.com",
+      phone: "12345678",
+      isDeleted: true,
+    })
```

Comment:
nit - might be better to move each of this into the respective test case in the "arrange" stage?

Reply:
nit - do we want to make the non-essential fields (name, userId, phone) to fallback to a default value in `setupUser` instead of having to specify them? doing so will make this setup cleaner and more readable since the "attention" will be on the `isDeleted` arg

Reply:
good points on both, have updated